pwndbg/commands/search.py

```diff
@@ -132,7 +144,20 @@ def print_search_hit(address) -> None:
 
 @pwndbg.commands.ArgparsedCommand(parser, category=CommandCategory.MEMORY)
 @pwndbg.commands.OnlyWhenRunning
-def search(type, hex, executable, writable, value, mapping_name, save, next, trunc_out) -> None:
+def search(
+    type,
+    hex,
+    executable,
+    writable,
+    interval,
+    limit,
+    aligned,
+    value,
+    mapping_name,
+    save,
+    next,
+    trunc_out,
```

Comment:
Just to be sure: the trunc_out still makes sense if we consider saving of results, right?

Reply:
Ya that was my thinking and why I left them separate. I suppose it may be worth documenting that in the usage somewhere to make the distinction clear?

Reply:
Yeah, ideally we should probably also add example usages in the docstring too

Reply:
(for that https://github.com/pwndbg/pwndbg/blob/dev/pwndbg/commands/vmmap.py#L46 would need to be used so that we can use newlines in the docstring heh)

Reply:
I added a couple small comments in the same structure as vmmap just commenting on the difference, but I think probably to add the full documentation for the commands with examples depending on how you want to do it would be a different pull request so not going todo it for now.